app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./PostContainer.css";
+import CommentSection from "../CommentSection/CommentSection";
+
+const PostContainer = props => {
+  return (
+    <div>
+      {props.posts.map(post => (
```

Comment:
we wanna map our data in app.js first and then make a post container for each post, that way we're really getting an example of a reusable component